### DISCUSSION TITLE
Whisper model duration limit

### DISCUSSION THREAD

**Discussion by Amirhfm** [deepgram/community#366] [General help] [answered]
Hey Everyone! Does anyone know how can I transcribe videos that other more than 20 mins using the whisper model? seems like there is a 20-minute limit for this model despite Nova being unlimited. I found the whisper model more accurate and want to use this to transcribe my videos. Is there any custom solution that Deepgram offers for this case?

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @Amirhfm, it's actually the case that Deepgram's own models (Nova, Enhanced, Base, etc) are capped at 10 minutes of processing time, while our managed Whisper is capped at 20 minutes ([docs](https://github.com/deepgram/stem/pull/1233)). Whisper is significantly slower than Nova (nearly 8 times slower in our [latest Nova-2 benchmarking](https://deepgram.com/learn/nova-2-speech-to-text-api)).

I understand that you're preferring Whisper's output. If you are currently submitting a video file, you will see a notable improvement by stripping the audio from your video as pre-processing, and only submitting the audio to Deepgram. That will enable us to process more audio with Whisper within the 20 minutes. Video files are larger and slower to process than audio files are, regardless of model.
